src/native/external/zlib-ng-version.txt

```diff
@@ -8,3 +8,5 @@ We have removed the following folders from our local copy as these files are not
 - zlib-ng/docs/
 - zlib-ng/test/
 - zlib-ng/arch/s390/self-hosted-builder/
+
+We have also applied the custom patches under the patches/zlib-ng/ folder.
```

Comment:
```suggestion
```

Reply:
Can you add a line to this file that says

> Apply https://github.com/dotnet/runtime/pull/105433

to describe the change to apply as a diff from zlib-ng?

Reply:
Sure can do.

Reply:
Done.